pkg/provider/kubernetes/gateway/fixtures/httproute/gatewayclass_with_unknown_controller.yml

```diff
@@ -4,7 +4,7 @@ apiVersion: gateway.networking.k8s.io/v1alpha2
 metadata:
   name: my-gateway-class
 spec:
-  controllerName: unkown.io/gateway-controller
+  controllerName: unknown.io/gateway-controller
```

Comment:
In principle this is a violation of DNS namespace. Instead one should use an `example` domain or something owned by traefik.